src/components/Cart/Cart.tsx

```diff
@@ -28,7 +33,9 @@ const Cart = () => {
         ))}
       {cartItems.length !== 0 && (
         <div className="btn-order__container">
```

Comment:
With styled components usually it's better to create a separate component instead of using classes.
You can also create a common folder of "base" components and extend from them like
```ts
const MyHeader = styled(BaseHeader)`
// new styles or override styles
`
```